content/en/docs/setup/windows/user-guide-windows-nodes.md

```diff
@@ -11,7 +11,7 @@ weight: 70
 
 The Kubernetes platform can now be used to run both Linux and Windows containers. One or more Windows nodes can be registered to a cluster. This guide shows how to:
 
-* Register a Windows node to the cluster
+* Register a Windows Server 2019 (or later) node to the cluster
```

Comment:
maybe we should simply say "Register a Windows node to the cluster. Make sure you are using a supported operating system" and then link to the section with the supported OS `#compute` is the link